cms/djangoapps/import_from_modulestore/tasks.py

```diff
@@ -20,16 +22,66 @@
 log = get_task_logger(__name__)
 
 
-@shared_task
-@set_code_owner_attribute
-def save_legacy_content_to_staged_content_task(import_uuid: str) -> None:
+class ImportCourseTask(UserTask):
```

Comment:
We don't need to use hard-coded naming with "Courses", as planed to import more than just courses in the future.